.github/workflows/app_build.yml

```diff
@@ -628,3 +698,102 @@ jobs:
         /bin/rm OpenStudioApplication-*${{ env.COMPRESSED_EXT }} || true
         /bin/rm OpenStudioApplication-*${{ env.BINARY_EXT }} || true
         ls OpenStudioApplication-* || true
+
+
+  test_package:
```

Comment:
Is this guaranteed to run on the same machine as the build above for non-self-hosted runners?

Reply:
Or it doesn't matter because you are re-downloading the artifact.

Reply:
The idea was the opposite. I wanted to do the testing bit on a separate machine. Which is now what I've done including for arm64 by using a GHA hosted macos-14 runner (which is arm64).

FWIW, I suppose at some point the overhead of maintaining the self hosted mac mini will make us switch to a a GHA hosted one, even though it has less CPU power.

Reply:
Yup totally agree, last I checked the MAC arm runner was either not available or really expensive or something.  It will be great when we can deprecate the self hosted one. 

Testing on clean runners makes a lot of sense.

Reply:
Different runners at least. The gha ones aren't vanilla at all, they have a lot of preinstalled dev tools.
The only clean one you can get is on linux because you can run on a vanilla docker image for ubuntu.
I've put in a suggestion to github to add vanilla runners but I don't expect they will